shopify/product-tagger-public-remix-ssr/.template/config.json

```diff
@@ -1,91 +1,55 @@
 {
   "version": "v1",
-  "setup": "Setup.md",
-  "introduction": "Intro.md",
+  "webflow": {
+    "title": "Automated Product Tagger",
+    "description": "Automate tagging products within a Shopify store inventory. This app tracks keywords within product descriptions and creates a tag for each keyword found.",
+    "content": "webflow/content.md",
+    "mainImage": "webflow/tagger.webp",
+    "wistiaId": "udboa27dyt",
+    "features": {
+      "shopify": true,
+      "frontend": true,
+      "models": true,
+      "actions": true
+    }
+  },
   "featuredModels": {
     "allowedTag": {
       "description": "Stores the tag keyword and the shop it is associated with.",
-      "featuredFields": [
-        "keyword",
-        "shop"
-      ]
+      "featuredFields": ["keyword", "shop"]
     },
     "shopifyProduct": {
       "description": "Synchronizes and stores product data from connected Shopify stores.",
-      "featuredFields": [
-        "title",
-        "body",
-        "tags"
-      ]
-    },
-    "shopifyShop": {
-      "description": "Keeps track of Shopify shops that your app has been installed on.",
-      "featuredFields": [
-        "name",
-        "syncs",
-        "shopOwner"
-      ]
+      "featuredFields": ["body", "tags"]
     }
   },
   "featuredActions": {
     "shopifyProduct": {
       "create": {
-        "description": "Reacts to Shopify's product/create events and copies the product data in your database, and then calls `applyTags()` defined in `shopifyProduct/utils.js`",
-        "tags": [
-          "edited"
-        ]
+        "description": "Reacts to Shopify's product/create events and copies the product data in your database, and then calls `applyTags` defined in `shopifyProduct/utils.js` (if the description has changed).",
```

Comment:
What's `(if the description has changed)`? Product description? What does the `applyTags` function do?

Also it might be nicer for the assistant by explicitly mentioning the full file path: `api/models/shopifyProduct/utils.ts`

(this comment also applies to the rest of the `update` model actionbelow)